app/models/application_form.rb

```diff
@@ -21,6 +21,7 @@ class ApplicationForm < ApplicationRecord
   MINIMUM_COMPLETE_REFERENCES = 2
   MAXIMUM_REFERENCES = 10
   EQUALITY_AND_DIVERSITY_MINIMAL_ATTR = %w[sex disabilities ethnic_group].freeze
+  ENGLISH_SPEAKING_NATIONALITIES = %w[GB IE].freeze
```

Comment:
I'm interpreting the designs verbatim here.

Should this also include American, Canadian, and Australian?

Reply:
cc: @tvararu @tijmenb  - leave this as GB/IE for now?

Reply:
I think this is reasonable, no reason to add the others just now. We can revise as needed!